html/browser.html

```diff
@@ -20,30 +20,90 @@
     <script id="mineral-observation-template" type="text/x-handlebars-template">
         <tr>
             <td>
-                {{#if mineral.mineral-name-plain}}<div><strong>Mineral Name:</strong> <a href="{{mineral.mineral-mindat-url}}" target="_blank"> {{mineral.mineral-name-plain}}</a></div>
-                {{else}} <div><strong>Mineral Name:</strong> N/A</div>
+                {{#if mineral-species.mineral-name-plain}}
+                    <div>
+                        <strong>Mineral Name:</strong>
+                        <a href="{{mineral-species.mineral-mindat-url}}" target="_blank">{{mineral-species.mineral-name-plain}}</a>
+                    </div>
+                {{else}}
+                    <div>
+                        <strong>Mineral Name:</strong>N/A
+                    </div>
                 {{/if}}
 
-                {{#if locality}}
 
-                <div> <strong>Locality: </strong><a href="{{locality.locality-mindat-url}}" target="_blank"> {{locality.locality-label}}</a></div>
+                <!--{{#if locality}}-->
+
                 <div>
-                    <strong>Latitutde:</strong>  {{locality.lat}}
-                    <strong>Longtitude:</strong>  {{locality.lon}}
-                    <a href="http://www.google.com/maps/place/{{locality.coordinates}}" target="_blank"> (View in Google Map)</a>
+                    <strong>Locality:</strong>
+                    <a href="{{locality.locality-mindat-url}}" target="_blank">{{locality.locality-label}}</a>
+                    <sup>{{mineral-specimen.age-determination-process.mindat-claims-existence.label}}</sup>
                 </div>
-
-                {{#if locality.min-age}}
-                    {{#if locality.max-age}}
-                        <div><strong>Age:</strong> {{locality.min-age}} - {{locality.max-age}} Ma</div>
+                <div style="font-style: italic; text-indent: 20px">
```

Comment:
Let's put this style info in a CSS style section and reference it via a named style